### ISSUE TITLE
Printer file F-specs are missing rec length value=in PRINTER() keyword

### ISSUE THREAD

**Issue by m-tyler** [BrianGarland/vscode-rpgfree#39]
I am using version 0.0.21 and I am seeing printer file conversion issues.  Text case is not important for me in this issue.  
***Note: I am not converting to Full Free format***

Before:
```
     FPRINT     O    F  132        PRINTER OFLIND(*INOD)
     FPRP40ZPR  O    F  132        PRINTER OFLIND(*INOF)
     F                                     USROPN
     F                                     FORMLEN(88)
     F                                     FORMOFL(88)
     F                                     PRTCTL(PRTDSZ:*COMPAT)
     FPRP40YPR  O    F  132        PRINTER OFLIND(*INOG)
     F                                     USROPN
     F                                     FORMLEN(88)
     F                                     FORMOFL(88)
     F                                     PRTCTL(PRTDSY:*COMPAT)
     FPRP40XPR  O    F  132        PRINTER OFLIND(*INOA)
     F                                     USROPN
     F                                     FORMLEN(88)
     F                                     FORMOFL(88)
     F                                     PRTCTL(PRTDSX:*COMPAT)
     FQSYSPRT   O    F  132        PRINTER OFLIND(*INOC)
     F                                     FORMLEN(88)
     F                                     FORMOFL(88)
     F                                     PRTCTL(PRTDSQ:*COMPAT)
```
After:
```
        Dcl-F PRINT PRINTER OFLIND(*INOD);
        Dcl-F PRP40ZPR PRINTER OFLIND(*INOF) USROPN FORMLEN(88) FORMOFL(88) PRTCTL(PRTDSZ:*COMPAT);
        Dcl-F PRP40YPR PRINTER OFLIND(*INOG) USROPN FORMLEN(88) FORMOFL(88) PRTCTL(PRTDSY:*COMPAT);
        Dcl-F PRP40XPR PRINTER OFLIND(*INOA) USROPN FORMLEN(88) FORMOFL(88) PRTCTL(PRTDSX:*COMPAT);
        Dcl-F QSYSPRT PRINTER OFLIND(*INOC) FORMLEN(88) FORMOFL(88) PRTCTL(PRTDSQ:*COMPAT);
```
Expected: (notice the Printer(132) keyword)
```
        Dcl-F Print Printer(132) Usage(*Output) Oflind(*INOD);
        Dcl-F Prp40Zpr Printer(132) Usage(*Output) Oflind(*INOF) Usropn Formlen(88) Formofl(88) Prtctl(Prtdsz:*COMPAT);
        Dcl-F Prp40Ypr Printer(132) Usage(*Output) Oflind(*INOG) Usropn Formlen(88) Formofl(88) Prtctl(Prtdsy:*COMPAT);
        Dcl-F Prp40Xpr Printer(132) Usage(*Output) Oflind(*INOA) Usropn Formlen(88) Formofl(88) Prtctl(Prtdsx:*COMPAT);
        Dcl-F Qsysprt Printer(132) Usage(*Output) Oflind(*INOC) Formlen(88) Formofl(88) Prtctl(Prtdsq:*COMPAT); 
```

**Comment by BrianBateman** [comment]
This is still an issue
 FQSYSPRT   O    F  132        Printer
Converted to:
        Dcl-F QSYSPRT PRINTER;

**Comment by BrianGarland** [comment]
will be fixed in 0.0.22
